mesheryctl/pkg/utils/error.go

```diff
@@ -337,6 +337,28 @@ func formatError(msg string, cmd cmdType) string {
 		return fmt.Sprintf("%s\nSee %s for usage details\n", msg, relationshipUsageURL)
 	case cmdRelationshipGenerateDocs:
 		return fmt.Sprintf("%s\nSee %s for usage details\n", msg, cmdRelationshipGenerateDocs)
+	case cmdComponent:
```

Comment:
why dont we extract the formatter in a function , right now formatting for each branch is same

Reply:
@alphaX86 ?

Reply:
I think it would be better to open a new issue.

Reply:
IMO it will be ok to open a new issue for this.

Reply:
IMO it will be ok to open a new issue for it.

Reply:
Agreed

Reply:
I'll open and proceed it